src/main/java/com/zufar/onlinestore/user/endpoint/UserEndpoint.java

```diff
@@ -0,0 +1,36 @@
+package com.zufar.onlinestore.user.endpoint;
+
+import com.zufar.onlinestore.user.api.UserApi;
+import com.zufar.onlinestore.user.dto.UserDto;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@Validated
+@RequestMapping(value = UserEndpoint.API_CUSTOMERS)
+public class UserEndpoint {
```

Comment:
change name for more specific. It is a bad practice to name class that expose and implement API as 'Endpoint'